14/umbraco-cms/extending/health-check/guides/contentsecuritypolicy.md

```diff
@@ -0,0 +1,50 @@
+# Health check: Content Security Policy (CSP)
```

Comment:
No need for adding "Health Check" here - let's keep it consistent with the other Health Check guides 💪 

```suggestion
# Content Security Policy (CSP)
```

Reply:
I updated the PR with your suggestion but 'Health check: ' is used in the other articles right?

For example: 

![image](https://github.com/user-attachments/assets/e92a99e1-91b3-408a-9d23-d31355049380)